code/modules/mob/living/carbon/human/species.dm

```diff
@@ -227,10 +229,15 @@
 		to_chat(prefs.parent, span_warning("You forgot to set your synthetic name in your preferences. Please do so next time."))
 
 /datum/species/proc/on_species_gain(mob/living/carbon/human/H, /datum/species/old_species)
+	for(var/newtrait in inherent_traits)
+		ADD_TRAIT(H, newtrait, SPECIES_TRAIT)
 	return
```

Comment:
usesless return, forgot the should call parent

Reply:
Fixe